### DISCUSSION TITLE
React-Bootstrap File Input with React-Hook-Form

### DISCUSSION THREAD

**Discussion by lalittolani** [react-hook-form/react-hook-form#1728] [General]
I am using React-Bootstrap file input with react hook form.
 <Form className="p-1" onSubmit={handleSubmit(onSubmit)}>
 <Form.Group as={Row}>
                <Form.Label column sm={3}>
                  PO Attachment File
                </Form.Label>
 <Form.File
                    id="custom-file-translate-html"
                    label="Upload A File"
                    data-browse="Browse Attachment"
                    custom
                  >
                    <Controller
                      as={<Form.File.Input as='input' isValid />}
                      name="po_file"
                      defaultValue={_fileObj}
                      
                      
                      control={control}
                      onChange={([event]) => {
                        // Place your logic here
                        //console.log(event.target);
                     
                        return event;
                      }}
                      rules={{
                        required: 'File is Required',
                      }}
                    />
                  </Form.File>
                  } />
                  {errors.po_file && (
                     
                      {errors.po_file.message ||
                        'No File is Selected. Please Upload a File'}
                     
                  )}

</Form.Group >
</Form>

I am getting error 

"TypeError: can't access property "Input", react_bootstrap__WEBPACK_IMPORTED_MODULE_1__.Form.File is undefined"

**Comment by bluebill1049** [comment]
can you share a codesandbox for this question above? keep the codesndbox small and focused.
